thirdparty/CMakeLists.txt

```diff
@@ -134,11 +134,7 @@ if(NGRAPH_PDPD_FRONTEND_ENABLE OR NGRAPH_ONNX_IMPORT_ENABLE)
         endif()
         find_package(Protobuf 3.9.0 REQUIRED)
         set(Protobuf_LITE_LIBRARIES protobuf::libprotobuf-lite)
-        if(NGRAPH_USE_PROTOBUF_LITE)
-            set(Protobuf_LIBRARIES ${Protobuf_LITE_LIBRARIES})
-        else()
-            set(Protobuf_LIBRARIES protobuf::libprotobuf)
-        endif()
+        set(Protobuf_LIBRARIES protobuf::libprotobuf)
```

Comment:
Since we are switching to the protobuf-lite is full protobuf needed here?

Reply:
just for symmetricity